src/views/Order/Address.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react'
+import {
+  Badge,
+  Button,
+  Card,
+  CardBody,
```

Comment:
Codacy found an issue: ['CardBody' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)